scripts/docker/ubuntu/bionic/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM ubuntu:bionic
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get -y install \
+        locales \
```

Comment:
Can you please sort the package list alphabetically? This makes it easier to determine, which packages are installed and also reduces the chance that we add duplicate entries here.